### PR TITLE
Update default vector level tiling sizes for ARM and RISC-V depthwise conv.

### PR DESCRIPTION
**Pull request by hanhanW**
It used x86 configurations, which causes big unrolling factors. Falls it to default which is more reasonable because tuning is not involved yet.

### CODE REVIEW

**Comment by hanhanW** [comment]
The regression and improvements are mostly noise. This will impact more on the pad + consumer path.

File: compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp [hunk 1]
```diff
@@ -1308,16 +1308,33 @@ static SmallVector<int64_t> getConvWorkgroupSizes(func::FuncOp entryPointFn,
   assert(succeeded(variantOp) && "ExecutableVariantOp not found");
 
   if (isX86(*variantOp) || isRISCV(*variantOp)) {
-    tileSizes = {1, 1, 8, vectorSize * 2, 1, 1, 8};
+    TypeSwitch<Operation *>(op.getOperation())
+        .Case<linalg::Conv2DNhwcHwcfOp>(
+            [&](auto op) { tileSizes = {1, 1, 8, vectorSize * 2, 1, 1, 8}; })
+        .Case<linalg::DepthwiseConv2DNhwcHwcOp>(
+            [&](auto op) { tileSizes = {1, 1, 8, vectorSize, 1, 3}; })
```
Comment (dcaballe) [inline compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp:1315]: This is also new, right? Should this lead to any performance change?

**Comment by dcaballe** [comment]
5% and 1% improvements on MobileNet and DeepLabV3 on top of #10437. Great!

**Review by dcaballe — APPROVED** [review]

File: compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp (hunk 1 above)
Comment (hanhanW) [reply compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp:1315]: good catch. I thought it's a refactor, but it turns out that the config is changed. It was `{1, 1, 8, vectorSize * 2, 1, 3}`. Let me revert the change.

Comment (dcaballe) [reply compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp:1315]: The new values improve performance on RISC-V so I guess we can keep it :)

Comment (hanhanW) [reply compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp:1315]: I missed the comment. let's keep the new values for RISC-V side. I'll update the PR title and the code!

**Pull request merged**